packages/salesforcedx-vscode-core/src/commands/auth/orgLoginWeb.ts

```diff
@@ -46,19 +46,19 @@ export interface DeviceCodeResponse {
   verification_uri: string;
 }
 
-export class OrgLoginWebContainerExecutor extends SfdxCommandletExecutor<AuthParams> {
+export class OrgLoginWebContainerExecutor extends SfCommandletExecutor<AuthParams> {
   protected showChannelOutput = false;
   protected deviceCodeReceived = false;
   protected stdOut = '';
 
   public build(data: AuthParams): Command {
-    const command = new SfdxCommandBuilder().withDescription(
+    const command = new SfCommandBuilder().withDescription(
       nls.localize('org_login_web_authorize_org_text')
     );
 
     command
       .withArg(CLI.ORG_LOGIN_DEVICE)
-      .withLogName('force_auth_device_login')
+      .withLogName('org_login_web_container')
```

Comment:
Where did you get this mapping from?  I found something different in https://github.com/forcedotcom/cli/blob/main/releasenotes/sfdx/README.md.
![Screenshot 2024-03-15 at 3 51 56 PM](https://github.com/forcedotcom/salesforcedx-vscode/assets/139700604/73697402-07a4-4d22-9cbd-f317980792bb)

Reply:
I know the command name is different, but I'm mapping it to the class name and using container as ID for CB stuff